readthedocs/core/static.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import division, print_function, unicode_literals
+
+from django.contrib.staticfiles.finders import FileSystemFinder
+
+
+class SelectiveFileSystemFinder(FileSystemFinder):
+
+    """
+    Add user media paths in ``media/`` to ignore patterns.
+
+    This allows collectstatic inside ``media/`` without collecting all of the
+    paths that include user files
+    """
+
+    def list(self, ignore_patterns):
+        ignore_patterns.extend(['epub', 'pdf', 'htmlzip', 'json', 'man'])
```

Comment:
We need `static` here (and hopefully there's no side effects). Otherwise when you run `collectstatic` multiple times, you end up with `media/static/static/static/...`.

Reply:
I was on the fence whether to add this -- as we also (unfortunately) have a `static/` in `readthedocs/` that I was curious if we'd skip. I'll give a test here.

Reply:
And it looks like you can't do `static/static`...

Reply:
The other answer is we use `--clear` in dev at least.

Reply:
Yup, you're right, multiple runs makes a nest of `static/` paths. I still see the files collected that I was worried about, so I think we're safe.